CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,8 @@
+<!--
+ Copyright Strimzi authors.
+ License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+-->
+
```

Comment:
Can't we just add these files (CODE_OF_CONDUCT.md, GOVERNANCE.md, MAINTAINERS.md etc.) to some ignore list? It will make it a lot easier if we ever need to update them in all XX repositories and can just copy files and not do something special for this repo. (if any ignoring for this would be possible, I would suggest to also use it for all MD files and all files which have nothing to do with code)

Reply:
Absolutely, the tool can be configured to ignore any type of file/directories if needed. I will update the config in the next commit to ignore .md files across the board.